tests/unit/relays/test_ipv6_preprocessing.py

```
@@ -0,0 +1,135 @@
+NOTE: These tests construct Relays() directly (full unpatched __init__) so the
+real _preprocess_template_data runs. Do NOT use
+TestSetupHelpers.create_test_relays_instance here — it patches
+_preprocess_template_data out.
+"""
+
+import unittest
+
+from allium.lib.relays import Relays
+
+
+def _make_relay(fingerprint, or_addresses):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_make_relay`

(ANN202)